### ISSUE TITLE
PrivateKeyInfo.bit_size and PublicKeyInfo.bit_size returns non-integer

### ISSUE THREAD

**Issue by thka2315** [wbond/asn1crypto#175]
I use Ubuntu Eoan with python3-asn1crypto 0.24.0-1.
It looks like PrivateKeyInfo.bit_size and PublicKeyInfo.bit_size returns non-integer.
I can see that *.byte_size returns a value with int() but the *.bit_size functions does not. The documentation in the function says it should return an integer.

print(certificate.public_key.bit_size) returns in my case "256.0"

Shouldn't it return only "256"?

Thanks for a great library!

**Comment by joernheissler** [comment]
Hello,
thanks for the report!

Can you confirm if `PrivateKeyInfo.bit_size` really returns a `float` (or other non-`int`)?
Only code path I can spot which returns a float is `PublicKeyInfo.bit_size` iff the algorithm is `ec`:
```python
            if self.algorithm == 'ec':
                self._bit_size = ((len(self['public_key'].native) - 1) / 2) * 8
```
`/` in py3 yields a float.
The code needs some work anyway because of #161.

Do you see any non-ints for non-ec keys?

**Comment by thka2315** [comment]
You are absolutely correct. I only see it for EC certificates. Certificates with RSA keys returns an integer.

**Comment by wbond** [comment]
This was fixed in c79ce16b194c5592ab2f45bb9f5052b614a607a1

**Comment by wbond** [comment]
This is part of the 1.4.0 release
